code/src/main/kotlin/com/expediagroup/sdk/lodgingconnectivity/graphql/extension/ApolloErrorExtension.kt

```diff
@@ -0,0 +1,9 @@
+package com.expediagroup.sdk.lodgingconnectivity.graphql.extension
+
+import com.apollographql.apollo.api.Error
+
+fun Error.toSDKError() =
+    com.expediagroup.sdk.lodgingconnectivity.graphql.model.response.Error(
```

Comment:
((**_Optional_**))
Can we have an alias for this `com.expediagroup.sdk.lodgingconnectivity.graphql.model.response.Error`?

Reply:
Done!

Reply:
Sorry for the confusion. I meant to have an alias for the original `Error` entity from the Apollo package. The added alias doesn't add a real value. You may rename the interface itself to the new name or ignore the aliasing suggestion.